lib/airbrake/sidekiq/retryable_jobs_filter.rb

```diff
@@ -29,7 +33,9 @@ def retryable?(job)
       end
 
       def max_attempts_for(job)
-        if job['retry'].is_a?(Integer)
+        if @max_retries
```

Comment:
Can this actually ever be `true`? Probably only when we initialise like this: `RetryableJobsFilter.new(max_retries: nil)`. That will look a bit confusing IMO.

In all other cases we will have the value defaulted to `DEFAULT_MAX_RETRY_ATTEMPTS`.

Reply:
Well, `@max_retries` will be nil by default if `max_retries` is not passed when creating the object, because it's specified in the parameter list of `initialize`.